keps/sig-storage/1488-csi-migration-gce-pd/README.md

```diff
@@ -40,6 +45,38 @@ used as as described in its parent KEP. For all other contents, please refer to
 
 Please refer to the [CSI Migration Production Readiness Review Questionnaire](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration#production-readiness-review-questionnaire).
 
+### Test Plan
```

Comment:
Can you move the Test Plan to the Design Details section?
https://github.com/kubernetes/enhancements/commit/a86942e8ba802d0035ec7d4a9c992f03bca7dce9

Reply:
Done